apps/site/src/views/Resources/components/BackendGroup/BackendGroup.tsx

```diff
@@ -0,0 +1,74 @@
+import Link from "next/link";
+
+import styles from "./BackendGroup.module.scss";
+import { Backend_Group_Props, tag } from "../../interfaces/interfaces";
+import Clear_Tape_Left from "@/assets/images/clear_tape_left.svg";
+import Clear_Tape_Right from "@/assets/images/clear_tape_right.svg";
+
+export default function Backend_Group({
+	card,
+	title,
+	description,
+	tagList,
+	tapeOrientation,
+	className,
+}: Backend_Group_Props) {
+	let tapePosition;
+	switch (tapeOrientation) {
+		case "left":
+			tapePosition = (
+				<img className={styles.left_tape} src={Clear_Tape_Left.src} />
+			);
+			break;
+		case "right":
+			tapePosition = (
+				<img className={styles.right_tape} src={Clear_Tape_Right.src} />
+			);
+			break;
+	}
+
+	return (
+		<>
+			{/* height hardcoded for 3 tags */}
+			<div
+				style={{
+					backgroundImage: `url(${card})`,
+					backgroundSize: "cover",
+					backgroundRepeat: "no-repeat",
+					backgroundPosition: "center",
+					width: "495px",
+					height: "510px",
+					position: "relative",
+					marginTop: "50px",
+				}}
+				className={`${className} ${styles.wrapper}`}
+			>
+				{tapePosition}
+				<div className={styles.text_flexbox}>
+					<h2 className={styles.title}>{title}</h2>
```

Comment:
Issue: h2 is already used for the Backend Resources heading, so resource headings should be one level below (h3).